jc3mpconfig.json

```diff
@@ -0,0 +1,151 @@
+        "DefaultValue": "20",
+        "EnumValues": {},
+        "Suffix":"players"
+    },
+    {
+        "DisplayName": "Password",
+        "Category": "Server Settings",
+        "Description": "Password to join the server.",
+        "Keywords": "password",
+        "FieldName": "password",
+        "InputType": "password",
+        "IsFlagArgument": false,
+        "ParamFieldName": "password",
+        "IncludeInCommandLine": false,
+        "DefaultValue": "",
+        "EnumValues": {}
+    },
+    {
+        "DisplayName": "Description",
+        "Category": "Server Settings",
+        "Description": "Displayed in the server browser.",
+        "Keywords": "description",
+        "FieldName": "description",
+        "InputType": "text",
+        "IsFlagArgument": false,
+        "ParamFieldName": "description",
+        "IncludeInCommandLine": false,
+        "DefaultValue": "Powered By Amp",
+        "EnumValues": {}
+    },
+    {
+        "DisplayName": "Tick Rate",
+        "Category": "Server Settings",
+        "Description": "Maxium FPS for the server (default is 60).",
+        "Keywords": "tickrate",
+        "FieldName": "tickrate",
+        "InputType": "number",
+        "IsFlagArgument": false,
+        "ParamFieldName": "maxTickRate",
+        "IncludeInCommandLine": false,
+        "DefaultValue": "60",
+        "Suffix":"ticks per second",
+        "EnumValues": {}
+    },
+    {
+        "DisplayName": "Name",
+        "Category": "Server Settings",
+        "Description": "Displayed in the server browser.",
+        "Keywords": "name",
+        "FieldName": "name",
+        "InputType": "text",
+        "IsFlagArgument": false,
+        "ParamFieldName": "name",
+        "IncludeInCommandLine": false,
+        "DefaultValue": "JC3MP-AMP",
+        "EnumValues": {}
+    },
+    {
+        "DisplayName": "Server Announce",
+        "Category": "Server Settings",
+        "Description": "Add a check here if you want to announce your server on the master server browser list.",
+        "Keywords": "Server Announce",
+        "FieldName": "announce",
+        "InputType": "checkbox",
+        "IsFlagArgument": false,
+        "ParamFieldName": "announce",
+        "IncludeInCommandLine": false,
+        "DefaultValue": "false",
+        "EnumValues": {
+         "False": "false",
+         "True": "true"
+        }
+    },
+    {
+        "DisplayName": "Required DLCs",
+        "Category": "Server Settings",
+        "Description": "Comma separated list of owned DLC IDs required to join.  [A list is available here](https://just-cause.mp/docs/general/information/Server%20Configuration) Leave this blank to not require any DLCs to join.",
+        "Keywords": "dlcs",
+        "FieldName": "requiredDLC",
+        "InputType": "text",
+        "IsFlagArgument": false,
+        "ParamFieldName": "requiredDLC",
+        "IncludeInCommandLine": false,
+        "DefaultValue": "",
+        "EnumValues": {}
+    }
+]
```

Comment:
For the settings, I'd recommend adding a "Placeholder" that matches the DefaultValue. If no default, an example of what should be placed there. It can really help with things like the Required DLCs where users might enter things wrong without seeing it first.

Reply:
added.